doc/dds-core-xtypes/README.md

```diff
@@ -5,74 +5,76 @@ Lightweight C++ implementation of dds-xtypes.
 Given the following IDL,
 
 ```c++
-struct Inner {
+struct inner {
```

Comment:
I don't understand why you change it to lower case, IDL user-defined types are usually to be in upper case. Please, can you recover the previous state.

Reply:
we talked about it. Anyway, this can be easly fixed